src/components/ManageVaults.tsx

```diff
@@ -111,7 +113,7 @@ const ManageVaults = () => {
     );
   } else if (!chainConnection) {
     popupContent = 'Connect your wallet to manage your vaults.';
-  } else if (vaults?.size === 0) {
+  } else if (vaults?.size === 0 || isSmartWalletProvisioned === false) {
```

Comment:
Consider having its own case message. Perhaps "You have no Agoric smart wallet yet."

Reply:
I considered that but I'm leaning towards keeping this as is. We already show the dialog, so it's not necessary to tell them twice, and it doesn't matter to them that there's no smart wallet until they actually try to create a vault, where they'll see the "Provision" button in that view.